expvar/influxvar.go

```diff
@@ -0,0 +1,24 @@
+// Encapsulates access to the top-level influx expvar.Map
+//
+// The init() method of this package guarantees that the map always exist.
+//
+// Callers can force the initialization of the package by using the
+// public Get() method.
+//
+// Originally created to resolve a circular dependency between the 'stats' package
+// and the influxdb.NewStatistics() function.
```

Comment:
Or, rather, the source code file that did contain the influxdb.NewStatistics() function.